DUI3/Speckle.Connectors.DUI/Models/DocumentModelStore.cs

```diff
@@ -10,44 +13,17 @@ namespace Speckle.Connectors.DUI.Models;
 /// </summary>
 public abstract class DocumentModelStore
 {
-  private ObservableCollection<ModelCard> _models = new();
-
-  /// <summary>
-  /// Stores all the model cards in the current document/file.
-  /// </summary>
-  public ObservableCollection<ModelCard> Models
-  {
-    get => _models;
-    protected set
-    {
-      _models = value;
-      RegisterWriteOnChangeEvent();
-    }
-  }
+  private readonly List<ModelCard> _models = new();
```

Comment:
We need to have public `Models` property for UI. Otherwise we cannot serialize `DocumentModelStore` as expected whenever `GetDocumentState()` is called in base bindings.

See the error in console

![image](https://github.com/user-attachments/assets/2b883a98-c316-4a9d-8cd7-fba98ed3f852)